Havit.Blazor.Components.Web.Bootstrap/TreeViews/Internal/HxTreeViewItemInternal.razor

```diff
@@ -42,19 +39,30 @@
     }
 </div>
 
-@if (this.IsExpanded.Value && hasChildren)
+@if (hasChildren)
 {
-    foreach (var item in children)
-    {
-        <HxTreeViewItemInternal TItem="TItem"
-                        Item="item"
-                        TitleSelector="this.TitleSelector"
-                        IconSelector="this.IconSelector"
-                        CssClass="@this.CssClass"
-                        CssClassSelector="this.CssClassSelector"
-                        ChildrenSelector="this.ChildrenSelector"
-                        Level="@(this.Level + 1)"
-                        ContentTemplate="this.ContentTemplate"
-                        OnItemSelected="this.OnItemSelected" />
-    }
-}
\ No newline at end of file
+    <HxCollapse
+        Id="@collapseId"
+        CollapseDirection="CollapseDirection.Vertical"
+        InitiallyExpanded="initiallyExpanded"
+        OnShown="HandleAnimationCompleted"
+        OnHidden="HandleAnimationCompleted">
```

Comment:
@hakenr `HxCollapse` closes or opens without animation when the `HxTreeViewItemInternal` isn't rerendered in between `HxCollapse` state changes. I haven't found the underlying issue yet.

Reply:
@hakenr could you please merge this branch, so that I can complete the other PR. I don't think this work-around causes any issues. Thanks

Reply:
Fixed the issue with StateHasChanged + merging.